src/transformers/models/bert/modeling_bert.py

```diff
@@ -251,7 +251,7 @@ def __init__(self, config, position_embedding_type=None):
             )
 
         self.num_attention_heads = config.num_attention_heads
-        self.attention_head_size = int(config.hidden_size / config.num_attention_heads)
+        self.attention_head_size = int(config.hidden_size // config.num_attention_heads)
```

Comment:
I think you change this just to trigger CI. Should revert before merge.

Reply:
Yap, it will be reverted for sure, thanks for flagging 👍🏻

Reply:
Done ✅